python/grass/jupyter/utils.py

```diff
@@ -104,7 +104,7 @@ def estimate_resolution(raster, mapset, location, dbase, env):
         flags="g",
         input=raster,
         mapset=mapset,
-        location=location,
+        project=location,
```

Comment:
My comment is about the function's header: In Python can you call positional arguments with their names as keywords ? If not, the whole function could be migrated to project.

Reply:
Oh, I never heard of that syntax, I'll keep my eye open if I see it! I know that in C# you can choose to name all arguments when you call a function, so the name rather than the order is important.